js/data/locale/bh/scripts.jf

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
I think Behari is still written in Devanagari, right? Why are these files deleted?